widgets/src/google/Area.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3/d3", "./CommonND"], factory);
+    } else {
+        root.Area = factory(root.d3, root.CommonND);
+    }
+}(this, function (d3, CommonND) {
+
+    function Area() {
+        CommonND.call(this);
+        this._class = "google_Area";
+
+        this._chartType = "AreaChart";
+    };
+    Area.prototype = Object.create(CommonND.prototype);
+    
+    Area.prototype.publish("areaOpacity", null, "number", "Font Size");
+    Area.prototype.publish("isStacked", false, "boolean", "Stacks the elements in a series");
+    
+    Area.prototype.getChartOptions = function () {
+        var retVal = CommonND.prototype.getChartOptions.apply(this, arguments);
+        //  TODO:  Add Area Properties Here
```

Comment:
i thought about putting it in common.js ... but its not common enough only for line/bar/column for both